R/data.R

```diff
@@ -32,15 +32,23 @@
 "adult_seeds"
 
 #' @title Proportion Hatchery
-#' @description The proportion of hatchery fish in each tributary, source: TODO
+#' @description The proportion of hatchery fish in each tributary
+#' @format 1 dimensional array [31 watersheds]
+#' @source 
+#' Proportion hatchery was derived using coded wire tag analysis. Details and methodology are available \href{https://cvpia-osc.github.io/fallRunDSM/articles/hatchery-analysis.html}{CVPIA Hatchery Analysis}
```

Comment:
> https://cvpia-osc.github.io/fallRunDSM/articles/hatchery-analysis.html


I know the link to the latefallrun version of this doesn't exists, but its best we change it to what we expect to be and for it remain broken as a reminder that we need to finish it